src/flow/ext_multiroute_flow.jl

```diff
@@ -0,0 +1,230 @@
+"""
+Computes the maximum multiroute flow (for any real number of routes)
+between the source and target vertexes in a flow graph using the
+[Extended Multiroute Flow algorithm](http://dx.doi.org/10.1016/j.disopt.2016.05.002).
+If a number of routes is given, returns the value of the multiroute flow as
+well as the final flow matrix, along with a multiroute cut if
+Boykov-Kolmogorov max-flow algorithm is used as a subroutine.
+Otherwise, it returns the vector of breaking points of the parametric
+multiroute flow function.
+Use a default capacity of 1 when the capacity matrix isn\'t specified.
+Requires arguments:
+- flow_graph::DiGraph                    # the input graph
+- source::Int                            # the source vertex
+- target::Int                            # the target vertex
+- capacity_matrix::AbstractArray{T,2}    # edge flow capacities
+- flow_algorithm::AbstractFlowAlgorithm, # keyword argument for algorithm
+- routes::Int                            # keyword argument for routes
+"""
+
+# EMRF (Extended Multiroute Flow) algorithms
+function emrf{T<:AbstractFloat,R<:Number}(
```

Comment:
Indeed, complex numbers would be problematic. I think it is fine for irrationals (even if it is unlikely to happen). I guess possible users could also convert to Float.

I will update to `R<:Real` if you're OK with it.